app/_components/companion-recruitment-filter.tsx

```diff
@@ -35,7 +40,38 @@ const CompanionRecruitmentFilter = () => {
   const ageRef = useRef([20, 30]);
   const personCountRef = useRef(1);
 
-  const handle = () => {};
+  const joinQuery = (...targets: string[][]) => {
+    return targets.map(target => target.join('&'));
+  };
+
+  const setObject = (regionQuery: string) => {
+    return {
+      gender: radioRef.current,
+      regions: regionQuery,
+      transportation: transportationRef.current,
+      startAge: ageRef.current[0],
+      endAge: ageRef.current[1],
+      totalPeople: personCountRef.current,
+    };
+  };
+
+  const getQuery = () => {
+    const [regionQuery] = joinQuery(
+      Object.keys(checkboxRef.current).filter(key => checkboxRef.current[key]),
+    );
```

Comment:
```suggestion
    const regionQuery = 
      Object.keys(checkboxRef.current).filter(key => checkboxRef.current[key]).join('&');
```

함수를 안만들어도 단순할수있을것 같아서 혹시 이런것은 어떻게 생각하시나요?!?🙂

Reply:
그러면 저는 util 로 분리해보는것도 좋을것 같긴합니다.!👍